cmd/config/config.go

```diff
@@ -12,15 +12,22 @@ import (
 //go:generate pflags Config
 
 var (
-	defaultConfig = &Config{}
-	section       = config.MustRegisterSection("root", defaultConfig)
+	DefaultLimit  int32 = 100
+	defaultConfig       = &Config{
+		Limit: DefaultLimit,
+	}
+	section = config.MustRegisterSection("root", defaultConfig)
 )
 
 // Config hold configration for flytectl flag
 type Config struct {
-	Project string `json:"project" pflag:",Specifies the project to work on."`
-	Domain  string `json:"domain" pflag:",Specified the domain to work on."`
-	Output  string `json:"output" pflag:",Specified the output type."`
+	Project       string `json:"project" pflag:",Specifies the project to work on."`
+	Domain        string `json:"domain" pflag:",Specified the domain to work on."`
+	Output        string `json:"output" pflag:",Specified the output type."`
+	FieldSelector string `json:"field-selector" pflag:",Specified the filter"`
+	SortBy        string `json:"sort-by" pflag:",Specified sort key"`
+	Limit         int32  `json:"limit" pflag:",Specified the limit"`
```

Comment:
Since we are adding a limit, we should support paginated queries too. 
Please create a separate issue so that we track it. 
Add a todo here with the issue no. aswell.